lukefi/metsi/data/formats/smk_util.py

```diff
@@ -137,25 +140,43 @@ def parse_drainage_category(source: str) -> int or None:
         return 5
     else:
         return util.parse_int(source)
+    
+def parse_development_class(source: str) -> int:
+    """ TODO: Waiting for future implementation.
+
+    At the moment (22.9.2023) development class variable is extracted
+    from .gpkg (not in .xml) format but not used by any model in simulation.
+    For that reason constant zero is returned.
+
+    :returns: zero """
+    return 0
 
```

Comment:
Don't leave stub TODO methods. Make a github issue instead for this "future implementation".